### PR TITLE
Clarity over supported devices

### PR DESCRIPTION
**Pull request by woter1832**
## Proposed change

Only a few devices are supported. Speakers are NOT supported. Change the wording to try and make it clearer.

## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -49,7 +49,7 @@ You are in control of the information and capabilities exposed to Home Assistant
 
 <div class='note'>
 
-The Nest Smart Device Management (SDM) API **requires a US$5 fee**.
+The Nest Smart Device Management (SDM) API **requires a US$5 fee**. BEFORE PAYING, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices). Speakers are NOT included.
```
Comment (frenck) [inline source/_integrations/nest.markdown]: Full capitalized words are generally considered to be screaming. Let's not do that, as there is no reason for it either. Additionally, suggesting to make it similar to the change made above:

```suggestion
The Nest Smart Device Management (SDM) API **requires a US$5 fee**. Before buying, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices).
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/nest.markdown [hunk 2]
```diff
@@ -49,7 +49,7 @@ You are in control of the information and capabilities exposed to Home Assistant
 
 <div class='note'>
 
-The Nest Smart Device Management (SDM) API **requires a US$5 fee**.
+The Nest Smart Device Management (SDM) API **requires a US$5 fee**. *Before buying*, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices). Nest speakers (Mini and Audio) are *not* currently included.
```
Comment (frenck) [inline source/_integrations/nest.markdown]: Removal of unneeded emphasis. I also removed the additional line. The link already provides the list of devices supported.

```suggestion
The Nest Smart Device Management (SDM) API **requires a US$5 fee**. Before buying, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices).
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/nest.markdown (hunk 1 above)
Comment (woter1832) [reply source/_integrations/nest.markdown]: Hi @frenck, I've made an additional commit, using italics in lieu of capitals. (Although, if capitals = screaming, perhaps that's not a bad thing in this case).

Comment (frenck) [reply source/_integrations/nest.markdown]: Screaming is always bad. There if not much to discuss in that regard.

**Review by frenck — APPROVED** [review]
Thanks, @woter1832 👍

**Pull request merged**